docs: final documentation changes (#5080)
recognai/rubrix: docs/_source/_common/snippets/start_page.md

```python
import argilla as rg

client = rg.Argilla(
    api_url="<api_url>",
    api_key="<api_key>"
    # extra_headers={"Authorization": f"Bearer {HF_TOKEN}"}
)
```

### 3. Create your first dataset

Specify a workspace where the dataset will be created. Check your workspaces in ["My settings"](/user-settings). To create a new workspace, check the [docs](https://argilla-io.github.io/argilla/latest/how_to_guides/workspace/).

> Create a Dataset with two labels ("positive" and "negative"). Don't forget to replace "<your-workspace>". Here, we are using a task template, check the docs to [create a fully custom dataset](https://argilla-io.github.io/argilla/latest/how_to_guides/dataset/).

Reviewer: ```suggestion
Here, we are defining a creating a dataset with a text field and a label question ("positive" and "negative"), check the docs to [create a fully custom dataset](https://argilla-io.github.io/argilla/latest/how_to_guides/dataset/). Don't forget to replace "<your-workspace>". 
```

```python
settings = rg.Settings(
    guidelines="Classify the reviews as positive or negative.",
    fields=[
        rg.TextField(
            name="review",
            title="Text from the review",
            use_markdown=False,
        ),
    ],
    questions=[
        rg.LabelQuestion(
            name="my_label",
            title="In which category does this article fit?",
            labels=["positive", "negative"],
        )
    ],
)
dataset = rg.Dataset(
    name=f"my_first_dataset",
    workspace="<your-workspace>",
    settings=settings,
    client=client,
)
dataset.create()
```

### 4. Add records

> You canCreate a list with the records you want to add. Ensure that you match the fields with the ones specified in the previous step.

Reviewer: ```suggestion
You can create a list with records that you want to add. Ensure that you match the fields with those specified in the question settings.
```
